nxviz/utils.py

```diff
@@ -130,4 +131,6 @@ def num_discrete_groups(data_container):
     'Accent_8': qualitative.Accent_8,
     'continuous': sequential.YlGnBu_9,
     'diverging': diverging.RdBu_11,
+    'weights': cubehelix_palette(50, hue=0.05, rot=0, light=0.9, dark=0,
```

Comment:
Nice! I like the suggested color palette.

Reply:
Yes this way even weights close to 0 are visible! Which is needed for NUPACK plots.